inc/taxonomies-functions.php

```diff
@@ -2243,3 +2248,37 @@ function taxopress_show_all_cpt_in_archive_result($request_tax){
 
             return $status;
 }
+
+/* Show taxonomy filter on post list */
+function taxopress_filter_dropdown( $taxonomy ) {
+	wp_dropdown_categories(
+		array(
+			'show_option_all' => sprintf( __( 'All %s', 'admin-taxonomy-filter' ), $taxonomy->label ),
```

Comment:
All translation strings should make use of taxopress translation to make it possible for new string translation. As in this sample `'admin-taxonomy-filter'` is not taxopress translation key